targets/psa_switch/psa_switch.cpp

```diff
@@ -422,6 +461,11 @@ PsaSwitch::egress_thread(size_t worker_id) {
     phv->get_field("psa_egress_input_metadata.parser_error").set(
         packet->get_error_code().get());
 
+    phv->get_field("psa_egress_input_metadata.max_recirculations").set(
```

Comment:
I am not sure it is a good idea to place these fields into existing standard PSA headers. These headers are well-defined and IMO we should not put stuff there which is not part of the PSA specification.